### PR TITLE
Update tests

### PR DESCRIPTION
**Pull request by odolbeau**
Update & clean tests

### CODE REVIEW

File: Tests/Unit/Catalogue/Operation/ReplaceOperationTest.php [hunk 1]
```diff
@@ -11,12 +11,12 @@
 
 namespace Translation\Bundle\Tests\Unit\Catalogue\Operation;
 
+use PHPUnit\Framework\TestCase;
 use Symfony\Component\Translation\MessageCatalogue;
 use Symfony\Component\Translation\MessageCatalogueInterface;
-use Symfony\Component\Translation\Tests\Catalogue\MergeOperationTest;
```
Comment (odolbeau) [inline Tests/Unit/Catalogue/Operation/ReplaceOperationTest.php]: Tests classes are not shipped with symfony anymore.

File: composer.json [hunk 2]
```diff
@@ -51,8 +52,8 @@
         ]
     },
     "scripts": {
-        "test": "vendor/bin/simple-phpunit",
-        "test-ci": "vendor/bin/simple-phpunit --coverage-text --coverage-clover=build/coverage.xml Tests/Unit"
+        "test": "vendor/bin/phpunit",
```
Comment (odolbeau) [inline composer.json]: Tests were failing cause simple-phpunit was installing `phpunit/phpunit` 6.5.
`phpunit/phpunit` 8.4 is not in `required-dev` & used directly on travis.

**Comment by odolbeau** [comment]
Ready to be merged! :+1:

**Comment by XWB** [comment]
👍

**Pull request merged**